packages/medusa-file-minio/src/services/minio.js

```diff
@@ -12,6 +12,7 @@ class MinioService extends FileService {
     this.endpoint_ = options.endpoint
     this.s3ForcePathStyle_ = true
     this.signatureVersion_ = "v4"
+    this.sslEnabled_ = !!options?.ssl_enabled
```

Comment:
issue: Does not default to true even though documentation says so
```suggestion
    this.sslEnabled_ = options?.ssl_enabled ? options.ssl_enabled : false
```